### DISCUSSION TITLE
Java distributions

### DISCUSSION THREAD

**Discussion by cmoisei** [ScoopInstaller/Java#372] [General]
Hi, 

I am probably not the only one that is confused about the java versions/json distributions - what is the best way, to get jdk11, the latest.

I used openjdk11.json for a while but them I notices that the version didn't get upgraded. Asking for a friend here.

Thanks!

**Comment by HUMORCE** [comment]
https://snyk.io/jvm-ecosystem-report-2021/

AND
with Scoop you can install multiple versions of the JDK.

*if a program keeps upgrading, it means it has more bugs or features fixed and other what we dont know.*

**Reply by cmoisei** [reply]
I agree, I was using openjdk11 variant. But that seems to be stucked at 11.0.2-9 ... I dont want to use Amazon's. I want o stay on the independent path. I was asking for a friend, here, on scoop, friend that would know what's the AdoptOpenJDK  scoop setup.

**Comment by se35710** [comment]
Hi,

OpenJDK only releases updates for current major release, no matter if it is a LTS release or not. So instead of 11.0.3, 12 was released.

There are alternatives:
- Temurin (previously known as AdoptOpenJDK) builds OpenJDK binaries, and continue to build for LTS releases (8, 11, 17, etc)
- Zulu (from Azul), backport patches and continues to deliver updates from JDK 6 to the latest release.

Both uses the same JVM as OpenJDK (hotspot), while for example Semeru uses OpenJ9.

More detailed information can be found in Wikipedia: https://en.wikipedia.org/wiki/Java_version_history
